packages/loot-core/src/server/accounts/sync.js

```diff
@@ -479,45 +512,43 @@ export async function syncAccount(userId, userKey, id, acctId, bankId) {
       return result;
     });
   } else {
-    const acctRow = await db.select('accounts', id);
-
-    // Otherwise, download transaction for the last few days if it's an
-    // on-budget account, or for the past 30 days if off-budget
-    const startingDay = monthUtils.subDays(
-      monthUtils.currentDay(),
-      acctRow.offbudget === 0 ? 1 : 30,
-    );
+    // Otherwise, download transaction for the past 30 days
+    const startingDay = monthUtils.subDays(monthUtils.currentDay(), 30);
 
-    const { transactions } = await downloadTransactions(
+    const { transactions, startingBalance } = await downloadTransactions(
       userId,
       userKey,
       acctId,
       bankId,
       dateFns.format(dateFns.parseISO(startingDay), 'yyyy-MM-dd'),
     );
 
+    // if (!transactions.length) {
+    //   return {
+    //     added: [],
+    //     updated: []
+    //   };
+    // }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should we remove this?